Foundation/NSHost.swift

```diff
@@ -33,7 +44,21 @@ open class Host: NSObject {
         _type = type
     }
     
-    static internal let _current = Host(nil, .current)
+    static func currentHostName() -> String {
+        let hname = UnsafeMutablePointer<Int8>.allocate(capacity: Int(NI_MAXHOST))
+        defer {
+            hname.deinitialize()
+            hname.deallocate(capacity: Int(NI_MAXHOST))
+        }
+        let r = gethostname(hname, Int(NI_MAXHOST))
+        if r < 0 || hname[0] == 0 {
+            print("gethostname returned empty string. Will use localhost")
```

Comment:
Since this appears to be an internal function, why not change the return type to `String?` and return `nil` here? (Also, if it's intentionally internal, probably best to write `static internal func`.)